frontend/src/metabase/admin/settings/components/widgets/CustomGeoJSONWidget.jsx

```diff
@@ -274,7 +274,7 @@ const GeoJsonPropertySelect = ({ value, onChange, geoJson }) => {
       {Object.entries(options).map(([name, values]) => (
         <Option key={name} value={name}>
           <div>
-            <div>{name}</div>
+            <div style={{ textAlign: "left" }}>{name}</div>
```

Comment:
Align dropdown title to the left as instead of center.
We are using `style` as it is used in other places in the component (quick fix).